### PR TITLE
Remove screenshot line in the config from all the maps

### PR DESCRIPTION
**Pull request by MinetestSam**

### CODE REVIEW

File: hunting_season/map.conf [hunk 1]
```diff
@@ -17,5 +17,4 @@ chests.1.from = -109,-35,-109
 chests.1.to = 109,30,-1
 chests.2.n = 50
 chests.2.from = 110,-35,110
-chests.2.to = -110,30,0
-screenshot = 16_hunting_season.png
+chests.2.to = -110,30,0
```
Comment (ClobberXD) [inline hunting_season/map.conf]: Missing newline

File: plain_battle/map.conf [hunk 2]
```diff
@@ -11,5 +11,3 @@ name = Plain battle
 initial_stuff = default:pick_stone,default:sword_stone,shooter:pistol,default:torch 5
 r = 110
 license = CC BY-SA 4.0
-screenshot = 10_plain_battle.png
-license = CC BY-SA 4.0
```
Comment (ClobberXD) [inline plain_battle/map.conf]: lol, what? :eyes:

Comment (MinetestSam) [reply plain_battle/map.conf]: Yeah I think it happened due to the enhancement PR for Plain Battle map

Comment (ClobberXD) [reply plain_battle/map.conf]: Funny how we missed this, lol

**Pull request merged**